operations/finance.md

```diff
@@ -347,80 +396,52 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AccountingItems` | array of [Accounting item](finance.md#accounting-item) | optional | The accounting items. |
-| `CreditCardTransactions` | array of [Credit card transaction](finance.md#credit-card-transaction) | optional | The credit card payment transactions. |
+| `OrderItems` | array of [Order item](#order-item) | optional | The accounting items. |
+| `PaymentItems` | array of [Payment item](#payment-item) | optional | The accounting items. |
+| `CreditCardTransactions` | array of [Credit card transaction](#credit-card-transaction) | optional | The credit card payment transactions. |
```

Comment:
What would be considered a CreditCardTransaction and not a PaymentItem? cards processed through ecommerce? terminal? 

Would manually entered credit card payments that weren't processed through Mews' payment gateway sit under PaymentItems?

Reply:
Every CreditCardTransaction is a payment. Specifying the extent only loads additional data.